### ISSUE TITLE
Some question about pretrain swin Transformer. 

### ISSUE THREAD

**Issue by liyih** [yichen928/SparseFusion#26]
Hi yicheng,
Thanks for your awesome work. I want to know how do you pretrain the swin-T based camera branch. Could you give me some detail information, like what dataset it uses (imagenet or nuimage)? What more, I want to know the input scale of image during pretrain. If you could share me with your code of pretrain it would be better. Thanks!
Best

**Comment by yichen928** [comment]
Thanks for your interest.

the Swin-T backbone and FPN are further finetuned by us on nuImages from the COCO pretrained weights (https://download.openmmlab.com/mmdetection/v2.0/swin/mask_rcnn_swin-t-p4-w7_fpn_1x_coco/mask_rcnn_swin-t-p4-w7_fpn_1x_coco_20210902_120937-9d6b7cfa.pth). All the finetuning settings (input size, lr, epochs) are same as https://github.com/open-mmlab/mmdetection3d/blob/main/configs/nuimages/mask-rcnn_r50_fpn_1x_nuim.py

**Comment by liyih** [comment]
Thanks for your reply!

**Comment by liyih** [comment]
In Swin-T based Sparsefusion the input size of image is 448×800.  But in mask-rcnn_r50_fpn_1x_nuim.py the size is not that, so do I need to use size 448×800?

**Comment by liyih** [comment]
In Swin-T based Sparsefusion the input size of image is 448×800. But in mask-rcnn_r50_fpn_1x_nuim.py the size is not that, so do I need to use size 448×800?

**Comment by yichen928** [comment]
That does not matter. You do not need to resize.
